website/source/docs/acl/acl-system.html.md

```diff
@@ -27,16 +27,35 @@ At the highest level, there are two major components to the ACL system:
  Accessor ID which is used to name a token, and a Secret ID which is used as the bearer token used to
  make requests to Consul.
 
- ACL tokens and policies are managed by Consul operators via Consul's
-[ACL API](/api/acl/acl.html), [ACL CLI](/docs/commands/acl.html), or systems like
+For many scenarios policies and tokens are sufficient, but more advanced setups
+may benefit from additional components in the ACL system:
+
+ * **ACL Roles** - Roles allow for grouping a set of policies and service
+   identities into a reusable higher-level entity that can be linked with many
+   tokens. (Added in Consul 1.5)
+
+ * **ACL Service Identities** - Service Identities allow for an abbreviated
+   syntax for expressing a kind of policy needed for an token to be usable with
```

Comment:
kait: This could be reworded, but I'm feeling that the likening of service identities to policies is very helpful. I think if we could flush it out here it will be really helpful for users and we could use the language in the other docs.